earth_enterprise/BUILD_RHEL_CentOS.md

```diff
@@ -24,10 +24,19 @@ sudo yum install -y epel-release
 
 ### RHEL 6
 ```
+# For RHEL 6 Workstation:
 sudo subscription-manager repos --enable=rhel-x86_64-workstation-dts-2
+
+# For RHEL 6 Server:
+sudo subscription-manager repos --enable=rhel-server-dts2-6-rpms
+
+# For all RHEL 6 Editions:
 sudo subscription-manager repos --enable=rhel-6-server-optional-rpms
+sudo yum install https://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
```

Comment:
I use this URL: `https://dl.fedoraproject.org/pub/epel/epel-release-latest-${RED_HAT_VERSION_MAJOR}.noarch.rpm` where `$RED_HAT_VERSION_MAJOR` is 6 or 7.

I think it's better not to have processor architecture in the URL, since this should work on all of them.

Reply:
I think this option. I'll just update the note to say that it will work best if RHEL is updated to the latest version